compiler/test/suites/gc.re

```diff
@@ -71,33 +62,95 @@ describe("garbage collection", ({test, testSkip}) => {
     "(1, (3, 4))",
     "Maximum memory size exceeded",
   );
-  assertRunGC("oomgc2", 64, "(1, (3, 4))");
-  assertRunGC("oomgc3", 32, "(3, 4)");
+  assertRunGC("oomgc2", 64, "(1, (3, 4))", "");
+  assertRunGC("oomgc3", 32, "(3, 4)", "");
 
   // gc tests
   assertRunGC(
     "gc1",
     160,
     "let f = (() => (1, 2));\n       {\n         f();\n         f();\n         f();\n         f()\n       }",
+    "",
   );
   /* https://github.com/grain-lang/grain/issues/774 */
   assertRunGC(
     "gc3",
     1024,
     "let foo = (s: String) => void\nlet printBool = (b: Bool) => foo(if (b) \"true\" else \"false\")\n\nlet b = true\nfor (let mut i=0; i<100000; i += 1) {\n  printBool(true)\n}",
+    "",
   );
-  assertFileRunGC(
+  assertRunGCError(
     "fib_gc_err",
     1024,
-    "fib-gc",
```

Comment:
It's now been deleted. It's gone because now with the module header, the program concatenation we were doing before doesn't work now, so that code just exists here.